python_modules/dagster/dagster_tests/core_tests/graph_tests/test_graph.py

```diff
@@ -1072,3 +1092,33 @@ def repro_2():
 
     with pytest.raises(DagsterTypeCheckDidNotPass):
         repro_2.execute_in_process()
+
+
+def test_to_job_input_values():
```

Comment:
test providing values when graph doesn't map those names
test that direct values take precedent over input def default values